tasks/configure.yml

```diff
@@ -1,39 +1,53 @@
 ---
+
 ## ASSEMBLE CONFIG - FRONTEND
+
 - name: 'Create directory for the frontend'
-  file: path={{ etc_prefix }}/haproxy/frontends.d state=directory
+  file:
+    path: "{{ etc_prefix }}/haproxy/frontends.d"
+    state: directory
 
 - name: 'Empty the folder if not already empty'
   command: find {{ etc_prefix }}/haproxy/frontends.d -name *.cfg -exec rm -f {} \;
 
 - name: 'Build up the frontends'
-  template: src=frontend.cfg dest={{ etc_prefix }}/haproxy/frontends.d/{{ item.name }}.cfg
+  template:
+    src: "frontend.cfg"
```

Comment:
There are no side-effects. It's purely a question of style/semantics.

PyYAML ignores trailing spaces, so we don't have to worry about those:

```python
>>> import yaml
>>> text = '''foo:
...   bar: baz '''
>>> yaml.load(text)
{'foo': {'bar': 'baz'}}
```

> I may sound a bit old school, but I like it when the code explicitly says : I intend to put a text string here.

Exactly, that's why I think using quotes sparingly to call attention to special cases (`'yes'`, `'0755'`) is important. In many syntax highlighters quoted strings stand out from unquoted strings, so you can see where the author has pointed out something unusual.